src/GenericAccessRequestForm/FormSubmissionUI.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Result, Button } from 'antd';
+import { Link } from 'react-router-dom';
+
+const FormSubmissionUI = ({
+  specificFormInfo,
+  formSubmissionStatus,
+  setFormSubmissionStatus,
+  setReqAccessRequestPending,
+}) => (
+  <div className='generic-access-container'>
+    <div className='generic-access-form-container'>
+      {formSubmissionStatus.status === 'success' ? (
+        <Result
+          status={formSubmissionStatus.status}
+          title={specificFormInfo.resultTitle}
+          subTitle={specificFormInfo.resultSubtitle}
+          extra={[
+            <Link key='discovery' to={'/discovery'}>
+              <Button>Go To Discovery Page</Button>
+            </Link>,
```

Comment:
This portion needs to be configurable for the workspace access request form (both button text and redirect path)

Reply:
This has been updated. The FormSpecificConstants.ts has key value pairs for the form specific constants objects, 
```
successRedirectLink: 
successRedirectText: 
```

The FormSubmissionUI references these values now: 
```
 <Link
              key={specificFormInfo.successRedirectLink}
              to={specificFormInfo.successRedirectLink}
            >
              <Button>{specificFormInfo.successRedirectText}</Button>
</Link>
```